src/main/kotlin/me/modmuss50/mpp/platforms/discord/DiscordWebhookTask.kt

```diff
@@ -49,6 +50,32 @@ interface DiscordWebhookOptions {
     }
 }
 
+@Suppress("MemberVisibilityCanBePrivate")
+class MessageStyle internal constructor() : java.io.Serializable {
```

Comment:
This should be an interface or abstract class, using Gradle properties, let me know if you want me to make this change.